woocommerce/woocommerce-gutenberg-products-block
# Add documentation for Formatters

This PR adds some documentation for our `Formatters`


Fixes #4210

### How to test the changes in this Pull Request:

1. Read the new documentation from a third-party extension author's point of view.
2. Ensure all links in the docs work correctly.



### Changelog

> Add documentation for ExtendRestApi's formatter classes.
